python/cudf/cudf/pandas/scripts/conftest-patch.py

```diff
@@ -12404,6 +12242,60 @@ def pytest_unconfigure(config):
     "tests/extension/test_string.py::TestStringArray::test_memory_usage[string=string[python]-True]",
     "tests/frame/methods/test_info.py::test_memory_usage_empty_no_warning",
     "tests/indexes/multi/test_integrity.py::test_memory_usage",
+    # These test fail an isinstance check where cudf.pandas returns a
```

Comment:
If this is the issue, can we hold off adding them to skip list? The following PR of mine fixes just this issue: https://github.com/rapidsai/cudf/pull/20460/files#diff-81a50fe0877c20003a77288cbadef4aa11bb7cad4575063e8b3babeb0981e204R494

Reply:
Yeah, based on the diff it won't fix all of them (ie. I'm skipping more tests than your PR fixed). But I'll check the stragglers out

Reply:
> I think this is because the xfails set is static but it really changes between pandas versions. 

Or could be because of the changes in your pr + https://github.com/rapidsai/cudf/pull/20460 resolved more cases. 



> So there are tests in the xfail set that aren't relevant to the pandas version one runs?

That can be a possibility but as pandas cleans out tests from their side but all of the tests are run only on latest pandas version.